chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DimensionalDataManager.java

```diff
@@ -111,16 +111,16 @@
     private CertificationEditionDAO certEditionDao;
 
     @Autowired
-    public DimensionalDataManager(PrecacheableDimensionalDataManager precache,
-            CertificationBodyDAO certificationBodyDao, CertificationCriterionDAO certificationCriterionDao,
-            EducationTypeDAO educationTypeDao, AgeRangeDAO ageRangeDao, TestFunctionalityDAO testFuncDao,
-            TestStandardDAO testStandardDao, TestToolDAO testToolDao, TestProcedureDAO testProcedureDao,
-            TestDataDAO testDataDao, AccessibilityStandardDAO asDao, UcdProcessDAO ucdDao,
-            QmsStandardDAO qmsDao, TargetedUserDAO tuDao, DeveloperStatusDAO devStatusDao,
-            SurveillanceDAO survDao, UploadTemplateVersionDAO uploadTemplateDao, QuarterDAO quarterDao,
-            ProductDAO productDao, DeveloperDAO devDao, JobDAO jobDao, MacraMeasureDAO macraDao,
-            CQMCriterionDAO cqmCriterionDao, CertificationEditionDAO certEditionDao) {
-        this.precache = precache;
+    public DimensionalDataManager(CacheableDimensionalDataManager cacheableDimensionalDataManager,
+                                  CertificationBodyDAO certificationBodyDao, CertificationCriterionDAO certificationCriterionDao,
```

Comment:
This looks like some odd indentation. Is this standard?

Reply:
My editor aligns blocks like this by default. I can look to change it in the settings if it's not the desired format. Honestly, I think this is a newer addition from a recent IntelliJ update.